src/java/org/apache/cassandra/db/SinglePartitionReadCommand.java

```diff
@@ -1287,6 +1294,12 @@ public PartitionIterator execute(ConsistencyLevel consistency, ClientState state
         {
             return StorageProxy.read(this, consistency, requestTime);
         }
+
+        @Override
+        public DataRange dataRange()
+        {
+            throw new UnsupportedOperationException("We shouldn't need to access the data range on a group.");
+        }
```

Comment:
If there's a better way to manage the interfaces/inheritance here, I could avoid this silly override...?